create content platform strategy page (#1358)
sourcegraph/handbook: content/strategy-goals/strategy/enablement/content-platform/index.md

We envision driving two workstreams inside of the team, one focused on the more traditional markdown-driven content architecture (handbook, docs) and one around CMS-driven content (marketing, blog, learn). By looking at our platforms from a content-neutral way, we still believe there are many shared elements between all the sites and want to optimize for efficiency. Also, if/when needs of the handbook and docs sites diverge more drastically from the other sites to the point where it causes inefficiency in the team, we will treat it as a signal that it is time to evaluate whether to split the team.

This team will be successful if it can deliver on its customers’ needs to help them achieve their goals. Streamlining the prioritization and delivery of work while enabling self-service ability is essential. Ultimately, meeting the delivery commitments on time will determine the team’s success.

### Customers

The Content Platform team has the following customers:

1. Marketing department: The marketing, blog, and learn sites.
1. Content Platform: The handbook site
1. Product department: The Docs site

## Strategy and Plans

> The next six months are focused on better defining the Handbook as a product and team. While forming our team we will enhance existing functionality like discovering and editing content.

Reviewer: ```suggestion
In the next six months, we will focus primarily on improving the self-service ability of the marketing site and blog for the Marketing department.
```

We envision driving two workstreams inside of the team, one focused on the more traditional markdown-driven content architecture (handbook, docs) and one around CMS-driven content (marketing, blog, learn). By looking at our platforms from a content-neutral way, we still believe there are many shared elements between all the sites and want to optimize for efficiency. Also, if/when needs of the handbook and docs sites diverge more drastically from the other sites to the point where it causes inefficiency in the team, we will treat it as a signal that it is time to evaluate whether to split the team.

This team will be successful if it can deliver on its customers’ needs to help them achieve their goals. Streamlining the prioritization and delivery of work while enabling self-service ability is essential. Ultimately, meeting the delivery commitments on time will determine the team’s success.

### Customers

The Content Platform team has the following customers:

1. Marketing department: The marketing, blog, and learn sites.
1. Content Platform: The handbook site

>    Product department: The Docs site

Reviewer: ```suggestion
1. Product department: The Docs site
```

## Strategy and Plans

The next six months are focused on better defining the Handbook as a product and team. While forming our team we will enhance existing functionality like discovering and editing content.

### Goals

#### FY22 Q4

> -Establish a scalable process for marketing web requests for the Content Platform team - Until the sites are more self-service, the Content Platform team will make the changes on behalf of the marketing team.

Reviewer: ```suggestion
- Establish a scalable process for marketing web requests for the Content Platform team - Until the sites are more self-service, the Content Platform team will make the changes on behalf of the marketing team.
```

> 

Reviewer: ```suggestion
```

> - Create a hiring plan for 2 x full-time engineers and 1 x engineering manager
> - Establish a unified technical strategy to drive all content platforms forward
>   - We have unofficially already settled on Next.js as the framework that serves our needs. It currently powers the handbook and learn sites and is being used to implement the blog redesign.
>     Deliver the implementation of the redesigned blog
>   - The blog design rollout is almost complete, and the team is actively working on building the templates.
>   - Integrate headless CMS for content management of the blog

Reviewer: ```suggestion
- Establish a unified technical strategy to drive all content platforms forward
- Deliver the implementation of the redesigned blog
```

#### FY23 Q1

- Convert the marketing site to use our preferred tech stack
- Implement Contentful CMS on marketing site
- [Marketing website redesign](https://docs.google.com/document/u/0/d/1_LOCIwHGb8A42wqWjZzT6s75_aY6qFK4gMSHLa5YPPE/edit)

> ### Longer-term

Reviewer: ```suggestion
#### Longer-term
```
